app/styles/app.scss

```diff
@@ -26,6 +26,18 @@
 
 @import 'layout';
 
+// any dynamic sections of the site need to be hidden in percy to prevent false
+// positives in the diffs
+@media only percy {
```

Comment:
@mansona thanks for doing this, I wasn't sure if team / @locks were cool with them being hidden in Percy or wanted the tests to not be dynamic instead. lgtm! 👍

Reply:
I'm down with whatever yinz decide 💪

Reply:
@amyrlam @locks we are not able to test them in Percy because we cannot (currently) wait for HighCharts to be finished animating before taking the Percy snapshot.

All dynamic things should be hidden by default and if we want to add back in Percy checks later once we're able to do the work that's great 👍 we should not have long-running papercuts that make us want to turn off CI tools 😞

Reply:
sounds good to me!